rust_src/src/minibuf.rs

```diff
@@ -34,3 +34,17 @@ fn active_minibuffer_window() -> LispObject {
         LispObject::from_raw(unsafe { minibuf_window })
     }
 }
+
+/// Specify which minibuffer window to use for the minibuffer.
+/// This affects where the minibuffer is displayed if you put text in it
+/// without invoking the usual minibuffer commands.
+#[lisp_fn]
+pub fn set_minibuffer_window(window: LispObject) -> LispObject {
+    window.as_minibuffer_or_error(); // just for the checks
+
```

Comment:
I would prefer `w.is_minibuffer()` check rather than new `as_minibuffer_or_error` method.

Reply:
`is_minibuffer` is insufficient based on the C code. It does not check if it is a valid window, etc.